site/index.js

```diff
@@ -1,8 +1,10 @@
 var js = import("../pkg/md_collab.js");
 
+var md = document.getElementById("md_input");
+
+window.addEventListener("load", read_md_file("index.md"));
```

Comment:
Please use camelcase for function names otherwise the js linter will fail

Reply:
ok good point